01. Consegui subir o servidor
02. Estava responsável pelo 'html_bloco_de_lista_de_usuarios'
03. Tentei rodar './testes/testa.sh html_bloco_de_lista_de_usuarios'
04. Deu o erro "ModuleNotFoundError: No module named 'produto'" na linha 9 do arquivo _TST.py
05. Removi o include do arquivo 'produto'
06. O erro passou a ser "NameError: name 'gera' is not defined", linha 34 do mesmo arquivo
07. O termo 'gera' estava sem aspas, sendo interpretado como variável. Coloquei entre aspas
08. Passou a apresentar o erro "TypeError: gera() missing 1 required positional argument: 'ids'", linha 29 do mesmo arquivo
09. A função 'utils_testes.testa_gera_html()' espera 6 argumentos, mas recebia apenas 5. Inclui a variavel 'pretty'.
10. Por último, passou a dar o erro "NameError: name 'bloco_texto' is not defined", arquivo html_bloco_de_usuario_IMP.py, linha 11. não consegui analisar esse problema.

EXTRA: Segundo a implementação anterior, a função 'html_bloco_de_lista_de_usuarios_TST.testa()' SEMPRE chamava o método 'gera()' do modulo, sem usar o argumento 'funcao' que recebe (esse argumento está atrelado ao problema das linhas 06 e 07). Então, troquei de 'modulo.gera', que sempre chamava o método 'gera', por 'getattr(modulo, funcao)', que vai chamar o método passado como argumento, ao inves de sempre o 'gera'.